Clicker/Models/ListDiffableModels/Poll.swift

```diff
@@ -86,9 +84,22 @@ class Poll {
         self.answers = currentState.answers
         self.upvotes = currentState.upvotes
         self.state = updatedPollState ?? poll.state
-        self.selectedMCChoice = poll.selectedMCChoice
         self.startTime = poll.startTime
     }
+    
+    func getSelected() -> Any? {
+        if let userId = User.currentUser?.id, let selected = answers[userId] as? JSON {
+            switch questionType {
+            case .multipleChoice:
+                return selected.stringValue
+            case .freeResponse:
+                return selected.arrayValue.map { (json) -> Int in
+                    return json.intValue
+                }
+            }
+        }
+        return nil
```

Comment:
we should also maybe add a check for `let selected = answers[userId] as? String` I think b/c we had to do so in `userDidSelect`